hardware/Layout-Desing-Guide.md

```diff
@@ -0,0 +1,486 @@
+
+<p align="center">
+  <img src="files/Layout-Design-Guide/3.png">
+</p>
+
+<div align="center">
+
+<b>Figure 5.</b> 
+
+</div>
+
+**Note:** It is true only in the case of Series 1. In the case Series 2 capacitors should be connected on same sides of the transmissin line. 
+
+
+4. Use at least 0.5 mm separation between traces/pads to the adjacent GND pours in the area of the matching network. This technique will minimize the parasitic capacitance and reduce the detuning effects.
+
+<p align="center">
+  <img src="files/Layout-Design-Guide/4.png">
+</p>
+
+<div align="center">
+
+<b>Figure 6.</b> 
+
+</div>
+
+5. The area under the RF chip and the matching network (in 4-layer PCBs, this is the first inner layer beneath the top layer) should be filled with continuous ground metal as it will show good ground reference for the matching network and will ensure a good, low impedance return path to the RF chip’s ground as well. Board routing and wiring should not be placed in this region to prevent coupling effects with the matching network. It is also recommended that the GND return path between the GND vias of the TX/RX matching network and the GND vias of the RFIC paddle should not be blocked in any way; the return currents should see a clear, unhindered pathway through the GND plane to the back of the RFIC.
+
+<p align="center">
+  <img src="files/Layout-Design-Guide/gnd.png">
+</p>
+
+<div align="center">
+
+<b>Figure 7.</b>
+</div> 
+
+
+6. Use 50 Ω grounded coplanar lines where possible for connecting the antenna or the U.FL connector to the matching to reduce sensitivity to PCB thickness variation. This will also reduce radiation and coupling effects. Use many GND vias near the coplanar lines in order to further reduce radiation.
+
+<p align="center">
+  <img src="files/Layout-Design-Guide/IFA.png">
+</p>
+
+<div align="center">
+
+<b>Figure 8.</b> 
+</div> 
+
+***
+## 3.2. Layout Design Practices - Sub-GHz matching network
+
+The layout consistency for the 4-layer dual-band EFR32 PCB is shown in the following figure. The individual layers will be marked with the currently presented colours during this presentation. Let’s see the sub-GHz EFR32 layout design practices one by one.
+
+<p align="center">
+  <img src="files/Layout-Design-Guide/layers.png">
+</p>
+
+<div align="center">
+
+<b>Figure 9.</b> 
+
+</div>
+
+
+
+
+1. Use the shortest traces possible to connect the first RX matching network component with the RX input pins of the chip. The RX matching network is very sensitive to any extra parasitic inductance and parasitic capacitance, this way it is important to minimize the detuning effects by using as short traces as possible.
+
+
+<p align="center">
+  <img src="files/Layout-Design-Guide/1s_2.png">
+</p>
+
+<div align="center">
+
+<b>Figure 10.</b> 
+
+</div>
+
+
+2. In order to decrease the parasitic capacitance towards the ground, it is recommended to apply a keepout on all inner layers beneath the area of the RX matching network.
```

Comment:
"between RX pins" is not true. Parasitic capacitance is between the trace and GND.

Reply:
Hi Andrea,
    Your statement is also true. But the final impact of parasite capacitance is CLNA between Rx pins. You can take a look at final paragraph of AN923 page 20. You can ignore my comment here since your statement is also right.
Regards,
Jun